api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -200,6 +200,12 @@ private RdrResearcher toRdrResearcher(DbUser dbUser) {
                 .map(RdrExportEnums::sexAtBirthToRdrSexAtBirth)
                 .collect(Collectors.toList()));
       }
+      if (null != dbDemographicSurvey.getGenderIdentityEnumList()) {
+        researcher.setGender(
+            dbDemographicSurvey.getGenderIdentityEnumList().stream()
+                .map(RdrExportEnums::genderToRdrGender)
+                .collect(Collectors.toList()));
+      }
```

Comment:
nit: It takes a bit of getting used to, but I think it's cleaner to do
```
Optional.ofNullable(dbDemographicSurvey.getGenderIdentityEnumList())
    .map(g -> g.stream().map(RdrExportEnums::genderToRdrGender).collect(Collectors.toList()))
    .ifPresent(researcher::setGender);
```

`if`, `null`, and `!=` should generally be minimized IMO.

Reply:
@jaycarlton : Do you mind taking another look?